examples/lighting-app/efr32/src/AppTask.cpp

```diff
@@ -366,29 +366,28 @@ void AppTask::FunctionHandler(AppEvent * aEvent)
         if (!sAppTask.mFunctionTimerActive && sAppTask.mFunction == kFunction_NoneSelected)
         {
             sAppTask.StartTimer(FACTORY_RESET_TRIGGER_TIMEOUT);
-            sAppTask.mFunction = kFunction_StartThread;
+            sAppTask.mFunction = kFunction_StartBleAdv;
         }
     }
     else
     {
-        // If the button was released before factory reset got initiated, start Thread Network
-        if (sAppTask.mFunctionTimerActive && sAppTask.mFunction == kFunction_StartThread)
+        // If the button was released before factory reset got initiated, start BLE advertissement in fast mode
+        if (sAppTask.mFunctionTimerActive && sAppTask.mFunction == kFunction_StartBleAdv)
         {
             sAppTask.CancelTimer();
             sAppTask.mFunction = kFunction_NoneSelected;
-#if CHIP_ENABLE_OPENTHREAD
-            if (!chip::DeviceLayer::ConnectivityMgr().IsThreadProvisioned())
+
+            if (!ConnectivityMgr().IsThreadProvisioned())
             {
-                StartDefaultThreadNetwork();
-                EFR32_LOG("Device is not commissioned to a Thread network. Starting with the default configuration.");
+                // Enable BLE advertisements
+                ConnectivityMgr().SetBLEAdvertisingEnabled(true);
+                // Set with fast Advertissing.
```

Comment:
```suggestion
```
It's obvious what the code is doing.  We should either document why it's doing it, or not document at all....